tests/e2e/specs/settings/applications-passwords.test.js

```diff
@@ -0,0 +1,103 @@
+import {
+    visitAdminPage,
+    __experimentalRest as rest,
+} from '@wordpress/e2e-test-utils';
+
+async function getResponseForApplicationPassword() {
+    return await rest({
+        method: 'GET',
+        path: '/wp/v2/users/me/application-passwords'
+    });
+}
+
+async function createApplicationPassword(applicationName) {
+    await visitAdminPage('profile.php');
+    await page.waitForSelector('#new_application_password_name');
+    await page.type('#new_application_password_name', `${applicationName}`);
+    await page.click('#do_new_application_password');
+
+    await page.waitForSelector('.notice');
```

Comment:
Nit: It isn't really clear what notice we're waiting for here. What if there're some other notices happening at the same time? I don't really have the knowledge for this feature so I'm not sure if this is a legitimate concern though 😅.

Reply:
Yeah, this makes sense. I've now added a new selector to all the waiting for notice message like this `#application-passwords-section .notice`.

Reply:
There's something weird, though. When creating a new app password, the notice message has these class names `notice notice-success is-dismissible new-application-password-notice`.

But when I do `await page.waitForSelector('.new-application-password-notice');`, it results in a timeout error and the selector is not found 🤔. Do you have any ideas of where this could come from?

Reply:
You're using `createApplicationPassword` twice in the 'should not allows to create two applications passwords with the same name' test, but the second time around you're passing the same name, so it'll error. The '.new-application-password-notice' class is only present on the success notice, not the error one, so I'm guessing that's where the timeout is happening.

There are at least two ways we can approach this:

If `createApplicationPassword` is meant to successfully complete the password creation flow, then checking for the presence of '.new-application-password-notice' is the only meaningful option, because '#application-passwords-section .notice' will display in case of error too. But if this is just a helper to fill in the password creation form, then we needn't do any check in the function itself - we can instead do the most relevant check (for success or error) in the test itself, after running the function.

If we go with the first, then in the 'should not allows to create two applications passwords with the same name' test we should just repeat the password creation steps manually for the second time, instead of calling the function. 

In terms of test readability, I'm a little more inclined towards the second option, but fine with either.

Reply:
I just tested it and it works for successful creations of application passwords, but not for failures. That's because the failure notices don't have the `new-application-password-notice` class. `#application-passwords-section .notice` seems like a good enough solution for this 👍 .

Reply:
@kevin940726 did you see my comment above? I'm not sure we should just let `createApplicationPassword` fail silently when the password isn't created for some reason.